HumanNeverDie/Projects/Features/HistoryFeature/Sources/Calendar/View/AMDCalendarCommonViews.swift

```diff
@@ -0,0 +1,123 @@
+//
+//  AMDCalendarCommonViews.swift
+//  HistoryFeature
+//
+//  Created by Seulki Lee on 6/27/25.
+//
+
+import SwiftUI
+import DesignSystem
+
+public struct CalendarTitleView: View {
+  let title: String
+  let onTap: () -> Void
+
+  public init(title: String, onTap: @escaping () -> Void) {
+    self.title = title
+    self.onTap = onTap
+  }
+
+  public var body: some View {
+    HStack {
+      Button(action: onTap) {
+        HStack(spacing: 4) {
+          Text(title)
+            .amdFont(.xlargeBold)
+            .foregroundColor(Color.gray100)
+
+          Image(systemName: "chevron.down")
+            .foregroundColor(Color.gray50)
+        }
+      }
+      Spacer()
+    }
+    .padding(.horizontal)
+  }
+}
+
+public struct CalendarWeekdayTitleView: View {
+  let items: [WeekdayValue]
+  let columns: [GridItem]
+
+  public init(items: [WeekdayValue], columns: [GridItem]) {
+    self.items = items
+    self.columns = columns
+  }
+
+  public var body: some View {
+    LazyVGrid(columns: columns, spacing: 15) {
+      ForEach(items, id: \.id) { weekday in
```

Comment:
`ForEach` 사용 시 `Identifiable`을 채택하고 있을 경우, 내부적으로 id를 인식해요! 
아래처럼 생략하고 사용하면 좀 더 코드가 짧아집니다.

```suggestion
      ForEach(items) { weekday in
```

Reply:
감사합니다!! 그렇군요!! 배워갑니다!! 👍